gnosis/gnosis-spec.ini

```diff
@@ -326,12 +329,28 @@ proxy_storage:
     andBool TO =/=Int GAS_TOKEN
     andBool TO =/=Int REFUND_RECEIVER
     andBool GAS_TOKEN =/=Int REFUND_RECEIVER
```

Comment:
Don't we need to say {TO, GAS_TOKEN, REFUND_RECEIVER} are also distinct to {MASTER_COPY_ID, PROXY_ID, ORIGIN_ID}?

Reply:
Also, in https://github.com/runtimeverification/verified-smart-contracts/pull/56/files#diff-713cf585e24f9780c47fbb3f16f9de75R301
Don't we need to specify ORIGIN_ID as well?

like:
```
activeAccounts: SetItem(MASTER_COPY_ID) SetItem(PROXY_ID) SetItem(ORIGIN_ID)
```

Reply:
This function doesn't use these arguments as accounts. These 3 inequalities are actually not needed and I deleted them. I guess it's best to keep the specification most general.